tests/test_yaml_search_space/test_search_space.py

```diff
@@ -0,0 +1,159 @@
+        assert pipeline_space["param_float1"].lower == 0.00001
+        assert pipeline_space["param_float1"].upper == 0.1
+        assert pipeline_space["param_float1"].log is True
+        assert pipeline_space["param_float1"].is_fidelity is False
+        assert pipeline_space["param_float1"].default is None
+        assert pipeline_space["param_float1"].default_confidence_score == 0.5
+        assert isinstance(pipeline_space["param_int1"], IntegerParameter)
+        assert pipeline_space["param_int1"].lower == -3
+        assert pipeline_space["param_int1"].upper == 30
+        assert pipeline_space["param_int1"].log is False
+        assert pipeline_space["param_int1"].is_fidelity is True
+        assert pipeline_space["param_int1"].default is None
+        assert pipeline_space["param_int1"].default_confidence_score == 0.5
+        assert isinstance(pipeline_space["param_int2"], IntegerParameter)
+        assert pipeline_space["param_int2"].lower == 100
+        assert pipeline_space["param_int2"].upper == 30000
+        assert pipeline_space["param_int2"].log is True
+        assert pipeline_space["param_int2"].is_fidelity is False
+        assert pipeline_space["param_int2"].default is None
+        assert pipeline_space["param_int2"].default_confidence_score == 0.5
+        assert isinstance(pipeline_space["param_float2"], FloatParameter)
+        assert pipeline_space["param_float2"].lower == 3.3e-5
+        assert pipeline_space["param_float2"].upper == 0.15
+        assert pipeline_space["param_float2"].log is False
+        assert pipeline_space["param_float2"].is_fidelity is False
+        assert pipeline_space["param_float2"].default is None
+        assert pipeline_space["param_float2"].default_confidence_score == 0.5
+        assert isinstance(pipeline_space["param_cat"], CategoricalParameter)
+        assert pipeline_space["param_cat"].choices == [2, "sgd", 10e-3]
+        assert pipeline_space["param_cat"].is_fidelity is False
+        assert pipeline_space["param_cat"].default is None
+        assert pipeline_space["param_cat"].default_confidence_score == 2
+        assert isinstance(pipeline_space["param_const1"], ConstantParameter)
+        assert pipeline_space["param_const1"].value == 0.5
+        assert pipeline_space["param_const1"].is_fidelity is False
+        assert isinstance(pipeline_space["param_const2"], ConstantParameter)
+        assert pipeline_space["param_const2"].value == 1e3
+        assert pipeline_space["param_const2"].is_fidelity is True
+
+    test_correct_yaml_file("tests/test_yaml_search_space/correct_config.yaml")
+    test_correct_yaml_file(
+        "tests/test_yaml_search_space/correct_config_including_types" ".yaml"
+    )
+
+
+@pytest.mark.neps_api
+def test_correct_including_priors_yaml_file():
+    """Test the function with a correctly formatted YAML file."""
+    pipeline_space = pipeline_space_from_yaml(
+        "tests/test_yaml_search_space/correct_config_including_priors.yml"
```

Comment:
Could we perhaps make the location (`tests/test_yaml_search_space/`) a global or class variable for all such relative path imports?
(Same, please note for the next release)